src/hooks/useOperatorOrder.ts

```diff
@@ -15,21 +15,39 @@ type UseOrderResult = {
   error?: string
   isLoading: boolean
   errorOrderPresentInNetworkId: Network | null
-  forceUpdate: () => void
+  forceUpdate?: () => void
 }
 
-interface GetOrderResult {
-  order: RawOrder | null
+type SingleOrder = RawOrder | null
+type MultipleOrders = RawOrder[] | null
+
+interface GetOrderResult<R> {
+  order: R | null
```

Comment:
If you have `null` on both types above, do you need it here? Or vice-versa

Reply:
@alfetopito did not found a better way to do this, after discussion with @ramirotw we have concluded the following:
I've typed `GetOrderResult`, `SingleOrder` and `MultipleOrder` independently by flexibility, so `GetOrderResult` receives a generic which happens to be also `null`.

The question is that since [`strictNullChecks`](https://github.com/gnosis/gp-ui/blob/856-search-tx-every-network/tsconfig.json#L30) is activated in the `tsconfig`, [`GetOrderResult`](https://github.com/gnosis/gp-ui/blob/856-search-tx-every-network/src/services/helpers/tryGetOrderOnAllNetworks.ts#L9) requires an implicit typing in order since tryGetOrderOnAllNetworks returns [`null` _at line 54_](https://github.com/gnosis/gp-ui/blob/856-search-tx-every-network/src/services/helpers/tryGetOrderOnAllNetworks.ts#L54) but is not the generic **R** value and the compiler complains.

Maybe the logic could be rethought to organize the typing differently, let me know WDYT.

Reply:
Not a big deal, let's leave it as is then